src/formatter/progress_formatter_spec.ts

```diff
@@ -31,10 +31,11 @@ Feature: a
     Given a pending step
   Scenario: a4
     Given a passing step
-  Scenario: a5
-    Given a skipped step
-  Scenario: a6
-    Given an undefined step
+  Rule: ab
+    Example: a5
+      Given a skipped step
+    Example: a6
+      Given an undefined step
```

Comment:
Can we please make a new test for this vs combining into one? This was really for each step status and I think its useful to have rule as a separate test

Reply:
Done